### ISSUE TITLE
Issues with ggforest()

### ISSUE THREAD

**Issue by kravvaz** [kassambara/survminer#472]
### Expected behavior 
ggforest() is supposed to generate forest plot for Cox proportional hazards models
### Actual behavior
ggforest() does not function properly. It gives an error "undefined columns selected" (below)
###  
library("survminer")
model <- coxph( Surv(time, status) ~ sex + rx + adhere, data = colon )
ggforest(model)

Error in `[.data.frame`(cbind(allTermsDF, coef[inds, ]), , c("var", "level",  : 
  undefined columns selected
In addition: Warning message:
In .get_data(model, data = data) :
  The `data` argument is not provided. Data will be extracted from model fit.
### `session_info()`

````{R}
# please paste here the result of
devtools::session_info()
- Session info -----------------------------------------------------------------------------
 setting  value                       
 version  R version 3.5.3 (2019-03-11)
 os       Windows 7 x64 SP 1          
 system   x86_64, mingw32             
 ui       RStudio                     
 language (EN)                        
 collate  English_United States.1252  
 ctype    English_United States.1252  
 tz       America/Chicago             
 date     2020-05-20                  

- Packages ---------------------------------------------------------------------------------
 ! package          * version    date       lib source                               
   assertthat         0.2.1      2019-03-21 [1] CRAN (R 3.5.3)                       
   backports          1.1.5      2019-10-02 [1] CRAN (R 3.5.3)                       
   broom              0.5.3.9000 2020-04-20 [1] Github (tidymodels/broom@3c922d5)    
   callr              3.4.1      2020-01-24 [1] CRAN (R 3.5.3)                       
   checkmate        * 2.0.0      2020-02-06 [1] CRAN (R 3.5.3)                       
   cli                2.0.1      2020-01-08 [1] CRAN (R 3.5.3)                       
   colorspace         1.4-1      2019-03-18 [1] CRAN (R 3.5.3)                       
   cowplot            1.0.0      2019-07-11 [1] CRAN (R 3.5.3)                       
   crayon             1.3.4      2017-09-16 [1] CRAN (R 3.5.3)                       
   curl               4.3        2019-12-02 [1] CRAN (R 3.5.3)                       
   data.table         1.12.8     2019-12-09 [1] CRAN (R 3.5.3)                       
   desc               1.2.0      2018-05-01 [1] CRAN (R 3.5.3)                       
   devtools           2.2.1      2019-09-24 [1] CRAN (R 3.5.3)                       
   digest             0.6.23     2019-11-23 [1] CRAN (R 3.5.3)                       
   dplyr            * 0.8.4      2020-01-31 [1] CRAN (R 3.5.3)                       
   ellipsis           0.3.0      2019-09-20 [1] CRAN (R 3.5.3)                       
   fansi              0.4.1      2020-01-08 [1] CRAN (R 3.5.3)                       
   farver             2.0.3      2020-01-16 [1] CRAN (R 3.5.3)                       
   forcats            0.4.0      2019-02-17 [1] CRAN (R 3.5.3)                       
   forestplot       * 1.9        2019-06-24 [1] CRAN (R 3.5.3)                       
   fs                 1.3.1      2019-05-06 [1] CRAN (R 3.5.3)                       
   generics           0.0.2      2018-11-29 [1] CRAN (R 3.5.3)                       
   ggplot2          * 3.3.0      2020-03-05 [1] CRAN (R 3.5.3)                       
   ggpubr           * 0.2.4      2019-11-14 [1] CRAN (R 3.5.3)                       
   ggsignif           0.6.0      2019-08-08 [1] CRAN (R 3.5.3)                       
   glue               1.3.1      2019-03-12 [1] CRAN (R 3.5.3)                       
   gridExtra          2.3        2017-09-09 [1] CRAN (R 3.5.3)                       
   gtable             0.3.0      2019-03-25 [1] CRAN (R 3.5.3)                       
   km.ci              0.5-2      2009-08-30 [1] CRAN (R 3.5.3)                       
   KMsurv             0.1-5      2012-12-03 [1] CRAN (R 3.5.2)                       
   knitr              1.28       2020-02-06 [1] CRAN (R 3.5.3)                       
   labeling           0.3        2014-08-23 [1] CRAN (R 3.5.2)                       
   lattice            0.20-38    2018-11-04 [1] CRAN (R 3.5.3)                       
   lifecycle          0.1.0      2019-08-01 [1] CRAN (R 3.5.3)                       
   magrittr         * 1.5        2014-11-22 [1] CRAN (R 3.5.3)                       
   Matrix             1.2-15     2018-11-01 [2] CRAN (R 3.5.3)                       
   memoise            1.1.0      2017-04-21 [1] CRAN (R 3.5.3)                       
   munsell            0.5.0      2018-06-12 [1] CRAN (R 3.5.3)                       
   pillar             1.4.3      2019-12-20 [1] CRAN (R 3.5.3)                       
   pkgbuild           1.0.6      2019-10-09 [1] CRAN (R 3.5.3)                       
   pkgconfig          2.0.3      2019-09-22 [1] CRAN (R 3.5.3)                       
   pkgload            1.0.2      2018-10-29 [1] CRAN (R 3.5.3)                       
   prettyunits        1.1.1      2020-01-24 [1] CRAN (R 3.5.3)                       
   processx           3.4.1      2019-07-18 [1] CRAN (R 3.5.3)                       
   ps                 1.3.0      2018-12-21 [1] CRAN (R 3.5.3)                       
   purrr              0.3.3      2019-10-18 [1] CRAN (R 3.5.3)                       
   R6                 2.4.1      2019-11-12 [1] CRAN (R 3.5.3)                       
   Rcpp               1.0.3      2019-11-08 [1] CRAN (R 3.5.3)                       
   remotes            2.1.1      2020-02-15 [1] CRAN (R 3.5.3)                       
   rlang              0.4.4      2020-01-28 [1] CRAN (R 3.5.3)                       
   rprojroot          1.3-2      2018-01-03 [1] CRAN (R 3.5.3)                       
   rstudioapi         0.11       2020-02-07 [1] CRAN (R 3.5.3)                       
   scales             1.1.0      2019-11-18 [1] CRAN (R 3.5.3)                       
   sessioninfo        1.1.1      2018-11-05 [1] CRAN (R 3.5.3)                       
   stringi            1.4.5      2020-01-11 [1] CRAN (R 3.5.3)                       
   stringr            1.4.0      2019-02-10 [1] CRAN (R 3.5.3)                       
   survival         * 2.43-3     2018-11-26 [2] CRAN (R 3.5.3)                       
   survivalAnalysis * 0.1.2      2020-04-17 [1] CRAN (R 3.5.3)                       
 V survminer        * 0.4.6      2020-05-20 [1] Github (kassambara/survminer@e828309)
   survMisc           0.5.5      2018-07-05 [1] CRAN (R 3.5.3)                       
   testthat           2.3.1      2019-12-01 [1] CRAN (R 3.5.3)                       
   tibble             2.1.3      2019-06-06 [1] CRAN (R 3.5.3)                       
   tidyr              1.0.2      2020-01-24 [1] CRAN (R 3.5.3)                       
   tidyselect         1.0.0      2020-01-27 [1] CRAN (R 3.5.3)                       
   tidytidbits        0.2.2      2020-04-15 [1] CRAN (R 3.5.3)                       
   usethis            1.6.0      2020-04-09 [1] CRAN (R 3.5.3)                       
   utf8               1.1.4      2018-05-24 [1] CRAN (R 3.5.3)                       
   vctrs              0.2.2      2020-01-24 [1] CRAN (R 3.5.3)                       
   withr              2.1.2      2018-03-15 [1] CRAN (R 3.5.3)                       
   xfun               0.12       2020-01-13 [1] CRAN (R 3.5.3)                       
   xtable             1.8-4      2019-04-21 [1] CRAN (R 3.5.3)                       
   yaml               2.2.1      2020-02-01 [1] CRAN (R 3.5.3)                       
   zoo                1.8-7      2020-01-10 [1] CRAN (R 3.5.3)                       

[1] C:/Users/936067/Documents/R/win-library/3.5
[2] C:/Program Files/R/R-3.5.3/library

 V -- Loaded and on-disk version mismatch.

**Comment by Istalan** [comment]
I could not reproduce your problem, however i noticed a few things:

1) you, of course, need to also load survival package
2) I'm using R 3.6.3
3) It could be that the coxph object did not store the data, so try to to call 
     ggforest(model, data = colon)


I hope any of this helps

**Comment by simonpcouch** [comment]
Looks like this issue is due to breaking changes in the development version of broom. This should be fixed by #485. :-)

**Comment by mblue9** [comment]
Just encountered this error with R 4.0 and it's reported [here](https://stackoverflow.com/questions/62960189/ggforest-returns-error-error-in-data-framecbindalltermsdf-coefinds?newreg=2397e7d979da4721892dda9e202bd225)  and [here](https://www.reddit.com/r/Rlanguage/comments/htddxu/i_keep_getting_this_ggforest_error_please_help/). 

Downgrading to an older version of R (3.6.3) and installing an older version of Broom (0.5.6) seems to work around it. 

Thanks for the info on the upcoming fix @simonpcouch! Will be good to get ggforest working with R 4.0.

**Comment by kassambara** [comment]
fixed now with the merge #485 , thanks
